From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 25 Aug 2020 11:06:20 +0100
Subject: ARM: dts: bcm2711: Add mmc_ddr_1_8v where possible

Advertise mmc_ddr_1_8v on platforms that support 1.8V SD/MMC DDR52.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi-4-b.dts | 1 +
 arch/arm/boot/dts/bcm2711-rpi-cm4.dts | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
index 21b20e334b1a..5469f5ad4bb3 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
@@ -173,6 +173,7 @@
 /* EMMC2 is used to drive the SD card */
 &emmc2 {
 	vqmmc-supply = <&sd_io_1v8_reg>;
+	mmc-ddr-1_8v;
 	broken-cd;
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
index d4ea928e30ce..823aeeca393d 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
@@ -109,9 +109,10 @@
 	};
 };
 
-/* EMMC2 is used to drive the SD card */
+/* EMMC2 is used to drive the eMMC/SD card */
 &emmc2 {
 	vqmmc-supply = <&sd_io_1v8_reg>;
+	mmc-ddr-1_8v;
 	broken-cd;
 	status = "okay";
 };
